src/NJsonSchema.CodeGeneration.TypeScript/Templates/ConvertToClass.liquid

```diff
@@ -37,6 +39,8 @@ if ({{ Value }}) {
 {%         endif -%}
 {%     endif -%}
     }
+} else {
```

Comment:
Wouldnt it be better to use elseif nullvalue != undefined to avoid generating some unused code?

Reply:
Good point - updated.